sdk/tables/azure-data-tables/azure/data/tables/_table_service_client.py

```diff
@@ -69,7 +69,11 @@ def __init__(
         super(TableServiceClient, self).__init__(
             account_url, service="table", credential=credential, **kwargs
         )
-        self._client = AzureTable(self.url, pipeline=self._pipeline)
+        self._client = AzureTable(
+            self.url,
+            transport=self._config.transport,
```

Comment:
i don't thinkt his line is right, just pass in **kwargs and that should be enouh